src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -95,6 +102,20 @@ class IOUAmountPage extends React.Component {
         this.focusTextInput();
     }
 
+    componentWillUnmount() {
+        if (!this.unsubscribeAppStateSubscription) {
+            return;
+        }
+        this.unsubscribeAppStateSubscription();
+    }
+
+    dismissKeyboardWhenBackgrounded(nextAppState) {
+        if (!nextAppState.match(/inactive|background/)) {
+            return;
+        }
+        Keyboard.dismiss();
```

Comment:
👋 Coming from https://github.com/Expensify/App/issues/26505

`Keyboard.dismiss` was added as a fix https://github.com/Expensify/App/issues/6285#issuecomment-968204846 where in native, `showSoftInputOnFocus` does not work if the app is backgrounded, then foregrounded
This was causing https://github.com/Expensify/App/issues/26505 on Desktop
We resolved this by moving this logic to a platform-specific file, so it's called only on native platforms (since this bug is present only on native)

Reply:
I love that. Thank you for moving it to a platform-specific file!